src/webots/nodes/utils/WbSupervisorUtilities.cpp

```diff
@@ -503,7 +503,10 @@ WbNode *WbSupervisorUtilities::getProtoParameterNodeInstance(int nodeId, const Q
     mRobot->warn(tr("%1: node not found.").arg(functionName));
     return NULL;
   }
-  return static_cast<WbBaseNode *>(node)->getFirstFinalizedProtoInstance();
+  WbBaseNode *proto = static_cast<WbBaseNode *>(node)->getFirstFinalizedProtoInstance();
+  if (!proto)
+    mRobot->warn(tr("Cannot get the PROTO instance for node '%1' (type '%2').").arg(node->modelName(), node->nodeModelName()));
```

Comment:
```suggestion
  if (!proto) {
    if (node->modelName() != node->nodeModelName())
      mRobot->warn(tr("Cannot get the PROTO instance for node '%1' (derived from '%2').").arg(node->modelName(), node->nodeModelName()));
    else
      mRobot->warn(tr("Cannot get the PROTO instance for node '%1'.").arg(node->modelName()));
    
```

Reply:
What I wanted is actually `defName()`, so I changed the code according to this.

Reply:
Did it.